pytorch_lightning/accelerators/gpu.py

```diff
@@ -39,6 +39,7 @@ def setup_environment(self, root_device: torch.device) -> None:
             MisconfigurationException:
                 If the selected device is not GPU.
         """
+        super().setup_environment(root_device)
```

Comment:
with DDP it will now fail after `init_dist_connection`

Reply:
This is currently a no-op. The super implementation is empty.

Reply:
I am actually pointing to this: https://github.com/PyTorchLightning/pytorch-lightning/blob/8394770d4afa5480f881229b150ac44eaa8c41b0/pytorch_lightning/strategies/ddp.py#L145-L151